fiftyone/core/collections.py

```
@@ -12701,6 +12732,59 @@ def _serialize_value(field_name, field, value, validate=True):
     return field.to_mongo(value)
 
 
+def _validate_media_reference_write(
+    sample_collection, field_name, filepath_error_message
+):
+    root_field = field_name.split(".", 1)[0]
+    if root_field == "_media_reference":
+        raise AttributeError(
+            "The private '_media_reference' field cannot be edited"
+        )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)